eventuate-client-java-micronaut-jdbc/src/main/java/io/eventuate/javaclient/micronaut/jdbc/EmbeddedTestAggregateStoreFactory.java

```diff
@@ -14,38 +16,43 @@
 import io.eventuate.javaclient.jdbc.JdkTimerBasedEventuateClientScheduler;
 import io.micronaut.context.annotation.Factory;
 import io.micronaut.context.annotation.Primary;
-import io.micronaut.runtime.context.scope.Refreshable;
-import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
-import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
 
 import javax.inject.Singleton;
 import javax.sql.DataSource;
+import java.util.Collections;
 
 @Factory
 public class EmbeddedTestAggregateStoreFactory {
 
-  @Primary
-  public EventuateCommonJdbcOperations eventuateCommonJdbcOperations(EventuateJdbcStatementExecutor eventuateJdbcStatementExecutor) {
-    return new EventuateCommonJdbcOperations(eventuateJdbcStatementExecutor);
+  @Singleton
```

Comment:
This looks suspicious.
I don't think a library should define `PlatformTransactionManager` - Perhaps the tests should?